apps/infrastructure/package.json

```diff
@@ -13,15 +13,20 @@
   "dependencies": {
     "@codedazur/cdk-static-site": "*",
     "@codedazur/essentials": "*",
+    "@storybook/addon-essentials": "7.0.26",
+    "@storybook/addon-interactions": "7.0.26",
+    "@storybook/builder-vite": "7.0.26",
+    "@storybook/react": "7.0.26",
     "aws-cdk-lib": "^2.50.0",
     "constructs": "^10.1.147",
+    "eslint-config-next": "12.0.4",
     "storybook": "*",
     "website": "*"
   },
   "devDependencies": {
     "aws-cdk": "^2.50.0",
     "dotenv": "^16.0.3",
-    "ts-jest": "^29.0.3",
+    "ts-jest": "^27.0.3",
```

Comment:
I know this pr is pretty old but we shouldn't downgrade packages. Adding it as a reminer